### ISSUE TITLE
Twilio phone number

### ISSUE THREAD

**Issue by MrJarnould** [MrJarnould/textbloodsugar#6]
Is it possible to write a script which purchases a phone number from Twilio automatically when a diabetic signs up on the webapp?

**Comment by PJUllrich** [comment]
Yes: https://www.twilio.com/docs/api/rest/incoming-phone-numbers
